clearer quarkus version requirement listed in release notes?

i tried update jbang.dev to roq 2.1 and hit this:

```
Caused by: java.lang.IncompatibleClassChangeError: Expecting non-static method 'boolean io.quarkus.runtime.LaunchMode.isDev()'
	at io.quarkiverse.web.bundler.deplo
```

was on 3.24.5, moved to 3.33.1 and it seem to go away.

but would be great if https://iamroq.dev/docs/releases/ would list what version of quarkus is expected/required.

--- a/blog/content/docs/releases.adoc
+++ b/blog/content/docs/releases.adoc
@@ -7,6 +7,8 @@ include::../../includes/edit-contrib.adoc[]
 
 We aim to keep the API stable, but since Roq is still young, breaking changes may happen. The good news: almost all breakage will be caught during build or xref:advanced.adoc#test-site-generation[site generation tests].
 
+**Minimum Quarkus version required:** Quarkus ≥ 3.33.1.
+
 {description}
 
 == Roq 2.1

